theme/material/vaadin-menu-bar-item-styles.html

```diff
@@ -0,0 +1,21 @@
+<link rel="import" href="../../../vaadin-material-styles/typography.html">
+
+<dom-module id="material-menu-bar-item" theme-for="vaadin-context-menu-item">
+  <template>
+    <style>
+      :host([theme="menu-bar-item"]) [part="content"] {
+        display: flex;
+        align-items: center;
```

Comment:
The flex alignment seems to be a little bit corrupted at least on screenshots.

Reply:
Seems ok to me. Updated the visual test so that it would be easier to check:

<img width="530" alt="Screen Shot 2019-04-30 at 15 30 51" src="https://user-images.githubusercontent.com/10589913/56961759-4d07d400-6b5d-11e9-8af1-7dc89f8677a1.png">